go/vt/topo/locks.go

```diff
@@ -265,10 +266,8 @@ func (l *Lock) unlockKeyspace(ctx context.Context, ts *Server, keyspace string,
 	// Detach from the parent timeout, but copy the trace span.
 	// We need to still release the lock even if the parent
 	// context timed out.
-	// Note that we are not using the user provided RemoteOperationTimeout
-	// here because it is possible that that timeout is too short.
 	ctx = trace.CopySpan(context.TODO(), ctx)
-	ctx, cancel := context.WithTimeout(ctx, defaultLockTimeout)
+	ctx, cancel := context.WithTimeout(ctx, RemoteOperationTimeout)
```

Comment:
👍

Reply:
Unlock is just a single remote operation. It doesn't need more time than a RemoteOperationTimeout. getLockTimeout is the time for which we want to lock the shard for.